app/services/analytics.ts

```diff
@@ -436,7 +436,9 @@ export default class Analytics extends Service {
 
     async _sendDataciteView(): Promise<void> {
```

Comment:
Just realized, do we want to update the `_trackDownloadTask` with this logic as well?